### DISCUSSION TITLE
How can I add the geometry to the second reach when I have manually split a channel into two reaches

### DISCUSSION THREAD

**Discussion by sjib** [teksi/wastewater#374] [Q&A]
I have manually "split" a channel and added a second reach. 

![grafik](https://github.com/user-attachments/assets/161cea9d-e7b2-4279-8fb1-accb950d6b0a)

How can I add a geometry to that new reach? 
if I select the new reach in the table view of vw_tww_reach and use the 'Linienobjekt hinzufügen' Tool it creates a new reach. But I just want to add a geometry to an existing object!

![grafik](https://github.com/user-attachments/assets/0271196a-9e79-497a-b58f-149439043721)

@urskaufmann Any hint?

**Comment by cymed** [comment]
use "Teil hinzufügen"

**Reply by urskaufmann** [reply]
Add part : Could not add part. This layer does not support multipart geometries.

I think this will not work...

**Reply by ponceta** [reply]
As far as I understand the VSA model, this should be done on vw_tww_channel which would be an aggregation of vw_tww_reach and with multilinestring geometries (not existing yet).

    Lausanne and SIGE often use channels aggregations so it should be a good thing to have this.

**Reply by cymed** [reply]
> As far as I understand the VSA model, this should be done on vw_tww_channel which would be an aggregation of vw_tww_reach and with multilinestring geometries (not existing yet).
> 
>     Lausanne and SIGE often use channels aggregations so it should be a good thing to have this.

I suggest not to use aggregated channels for this, as it might lead to problems with disaggregation. Better to have a "split reaches" tool.

**Comment by urskaufmann** [comment]
my workaround (because there is no UI): create a second reach with wizard, then change the fk_wastewater_structure in the database to the channel of the first reach and delete the wastewater_structure-record (and channel-record) of the second reach in the database or in the table (this channel has then no more connected reach and can therefore not be selected on the map...
